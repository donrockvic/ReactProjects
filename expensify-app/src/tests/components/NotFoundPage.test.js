import React from 'react';
import ReactShallowRender from 'react-test-renderer/shallow';

import NotFoundPage from '../../components/404';
 

test('should render Not found page', () => {
    const renderer = new ReactShallowRender();
    renderer.render(<NotFoundPage/>);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});