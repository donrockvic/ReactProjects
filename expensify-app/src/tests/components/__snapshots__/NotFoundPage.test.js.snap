// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Not found page 1`] = `
<div>
  This is from 404!
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
